Add log10.md (#1082)
Codecademy/docs: content/cpp/concepts/math-functions/terms/log10/log10.md

---
Title: 'log10()'
Description: 'Returns the base-10 logarithm of the argument.'
Subjects:
  - 'Computer Science'
Tags:
  - 'Functions'
  - 'Arithmetic'
  - 'Methods'
CatalogContent:
  - 'learn-c-plus-plus'
  - 'paths/computer-science'
---

> The **log10()** function returns the base-_10_ logarithm of the argument.

Reviewer: ```suggestion
The **log10()** function returns the base-10 logarithm of the argument.
```

## Syntax

> ```cpp

Reviewer 2: ```suggestion
```pseudo
```

 log10(x)
```

> If the `x` parameter is less than or equal to zero, then a domain error will occur. If `x` is equal to zero, a pole error may occur where the result has reached the point of infinity and cannot be expressly returned.

Reviewer 2: ```suggestion
If the `x` parameter is negative (less than zero), then a domain error will occur. If `x` is equal to zero, a pole error may occur where the result has reached the point of infinity and cannot be expressly returned.
```

> If the `x` parameter is less than or equal to zero, then a domain error will occur. If `x` is equal to zero, a pole error may occur where the result has reached the point of infinity and cannot be expressedly returned.

Reviewer: ```suggestion
If the `x` parameter is less than or equal to zero, then a domain error will occur. If `x` is equal to zero, a pole error may occur where the result has reached the point of infinity and cannot be expressly returned.
```

## Example

> The following example uses the `log10()` function to find the natural log of `5`:

Reviewer: ```suggestion
The following example uses the `log10()` function to find the base-10 log of `5`:
```

```cpp
#include <iostream>
#include <cmath>
using namespace std;

int main() {
  double x = 5;
  double result;
  result = log10(x);
  cout << "The log to the base of 10 of " << x << " is " << result << "\n";
  // Output: The log of 5 is 0.69897
}
```

## Codebyte Example

> The following example is runnable and returns the natural log of `10`:

Reviewer: ```suggestion
The following example is runnable and returns the base-10 log of `10`:
```
